spyder/plugins/profiler/widgets/profilergui.py

```diff
@@ -516,6 +517,21 @@ def hide_diff_cols(self, hide):
 
     def save_data(self, filename):
         """Save profiler data."""
+        # Save the profiler data using a stream in a csv file
+        stream = io.StringIO()
+        self.stats1[0].stream = stream
+        self.stats1[0].print_stats()
+        data = stream.getvalue()
+        data = 'ncalls' + data.split('ncalls')[-1]
+        data = '\n'.join([','.join(
+            line.rstrip().split(None, 6)) for line in data.split('\n')])
+
+        # # save it to disk

```

Comment:
double `#`, also please start comments with uppercase

Reply:
is the uppercase comments something general for Python, or it's Spyder specific?